### ISSUE TITLE
AgentDVR not accessable on ATT/Android

### ISSUE THREAD

**Issue by RaptorPCS** [ispysoftware/iSpy#263]
Have a business license and am using Cloudflare tunnels as a reverse proxy. On my iPhone and on any other device (Windows, MacOS, Android tablet), I am able to login and use the remote feature. On an Android Samsung S23 Ultra (and S22 Ultra), I get the login screen. However, after logging in, I get the loading screen followed by a message saying, "Try another browser or check the firewall isn't blocking ports. See server menu locally for remote access." The machine is a fresh install of Windows 10 with no firewall and no Windows security enabled. (For troubleshooting purposes, I don't plan to run it like that in production.) The same issue occurred when we had open ports to the internet, a domain forwarding to our network, and an SSL Cert from LetsEncrypt.

I'm at a loss. I've tried everything I could think of to fix this issue. Remote access works perfectly on several devices, but not on these specific ones. The only discernible difference is that the non-working devices are on AT&T. I even tried reducing playback to 480p in case it was a bandwidth restriction. Using a VPN (Cloudflare Warp) didn't help either. Here's the log I get when trying to login: 

<html><body>

4:39:30 PM | Process: Not private: 2600:387:b:3::91
-- | --
4:39:31 PM | Process: admin logged in from 2600:387:b:3::91
4:39:50 PM | Process: Not private: 2600:387:b:3::91
4:39:50 PM | Process: admin logged in from 2600:387:b:3::91
4:40:49 PM | TmrBroadcastElapsed: Removed 1 rtc sessions
4:41:07 PM | TmrBroadcastElapsed: Removed 1 rtc sessions

</body>
</html>

I see the addresses are IPv6 when the devices have IPv4 and v6 and the network does have v6 as well (even tried disabling it as well as under a different designation 

<html><body>

3:35:59 PM | Process: Not private: 2a07:23c0:9:a001::bfe3
-- | --
3:35:59 PM | Process: admin logged in from 2a07:23c0:9:a001::bfe3
3:36:53 PM | Process: Not private: 2a07:23c0:9:a001::bfe3
3:37:16 PM | TmrBroadcastElapsed: Removed 1 rtc sessions
3:37:19 PM | Process: Not private: 2a07:23c0:9:a001::bfe3
3:37:33 PM | Process: admin logged in from 2a07:23c0:9:a001::bfe3

</body>
</html>

It seems it might be a IPv6 issue? is there a way to force v4 only sign in on agent? I have also tried just opening the ports and using our IP directly with SSL off and it also does not login. I am so lost.

**Comment by ispysoftware** [comment]
What web browser are you using? Have you tried different browsers?

https://about.att.com/sites/broadband/network#:~:text=For%20example%2C%20as%20described%20in,%2C%20445%2C%20and%201900).

Looks like AT&T are blocking port 50001 which is one of the ports needed. 
https://www.ispyconnect.com/userguide-agent-DVR.aspx#remote

you can stop Agent, edit Media/XML/config.xml and change the ports the Turn server listens on:

TurnServerPort 3478 
TurnServerMinPort 50000
TurnServerMaxPort 50010

maybe try 3478,50020,50030 or something like that

Then restart it - make sure you use ports outside of the ranges of those blocked by AT&T - I'm guessing there's a few other ports they're blocking that aren't on that list.  - check it still works from other places

Also, call AT&T and tell them they're idiots :)

**Comment by RaptorPCS** [comment]
I edited config to the ones recommended and now am sitting on a infinite loading screen
I did try it in every browser (Firefox, chrome, Samsung internet). I get the same logs (user signs in and then removed rtc sessions) I have tried this on HTTP and HTTPS I am at a loss. the fact that I get further tells me it is something with ATT which sucks

**Comment by ispysoftware** [comment]
What did you change the ports to? Maybe try contacting ATT and ask them if either they can unblock those ports for you or suggest alternatives?

FWIW you should be able to use the subscription option to access it as it will be going via our servers instead

**Comment by RaptorPCS** [comment]
I did the ports recommended 50020-50030 I am considering the subscription but we already did the business license and wanted the most self hosted option. ill weigh my options. Im pretty sure ATT would try to sell a business plan or something like that which probably end up costing as much as the subscription thru you guys and Id rather support yall then a large corp like them. I did want to try to see if a VPN would be a better option first but id have to set all that up lol. Thank you for your support ill close this now. Thank you for confirming im not crazy ha.
